zenoh/path.py

```diff
@@ -18,8 +18,10 @@ def __init__(self, path):
         self.__path_regex = re.compile('^[^?#*]+$')
         if not self.is_valid(path):
             raise ValueError("Invalid Path: {}".format(path))
-        self.path = path
-
+        self._path = path
+    @property
+    def path(self):
+        return self._path
     @staticmethod
```

Comment:
```suggestion

    @staticmethod
```
Same here: add an empty line.

Reply:
@Harmouch101 : Unfortunately this commit failed to pass the Eclipse ECA check, because you didn't sign it (-S option with git command). If you committed via the Github page, please check that you added your GPG key to your Github profile.